src/events.rs

```diff
@@ -2,6 +2,17 @@ use std::path::PathBuf;
 
 #[derive(Clone, Debug)]
 pub enum Event {
+    WindowEvent {
+        window_id: usize,
```

Comment:
Perhaps this should be it's own `Id` type defined in the `window` module? That way we could also document it, specifying that every window will have a unique identifier associated with it for the duration of its lifetime? Maybe this is fine though, I'm not sure.